tests/test_logging.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+from mock import Mock
+
+from inspire_utils.logging import log_error_with_extra
+
+
+def test_log_error_with_extra_enables_the_stack_flag_if_it_is_not_disabled():
+    mocked_logger = Mock()
+
+    log_error_with_extra(mocked_logger, 'foo')
+    mocked_logger_error_kw_call_args = mocked_logger.error.call_args[1]
+
+    assert mocked_logger_error_kw_call_args == {'extra': {'stack': True}}
+
+
+def test_log_error_with_extra_does_not_edit_the_extra_stack_flag_if_it_is_provided_in_kw_args():
+    mocked_logger = Mock()
+
+    log_error_with_extra(mocked_logger, 'foo', extra={'stack': False})
+    mocked_logger_error_kw_call_args = mocked_logger.error.call_args[1]
+
+    assert mocked_logger_error_kw_call_args == {'extra': {'stack': False}}
```

Comment:
We can avoid these tests: they are just change-detector tests (https://testing.googleblog.com/2015/01/testing-on-toilet-change-detector-tests.html).

Reply:
Correct. I will remove them

Reply:
The first one I can agree, the second test I don't agree, it's making sure that the behavior of the function is to not override the keyword parameter passed. It makes sure that if you change anything in the function you will not override that parameter (something that already happened in a previous approach).
I don't see a clear line there.